redisinsight/api/src/modules/database/middleware/connection.middleware.ts

```diff
@@ -47,7 +46,7 @@ export class ConnectionMiddleware implements NestMiddleware {
     return {
       timeout: req.body?.timeout,
       instanceIdFromReq: req.params?.id,
-     };
+    };
   }
 
```

Comment:
In general I'm fine with such changes. You found bad code style and fixed it. But I think it should be in a different pr or in a smaller one. Anyway, thanks!

Reply:
Got it. I did only after conversation with @romansergeenkosofteq (He told that it is not a problem to refactor other parts in PR)

Reply:
BTW i will create another PR next time